src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -34,6 +36,30 @@ public class AddressBookParser {
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
     private static final Logger logger = LogsCenter.getLogger(AddressBookParser.class);
 
+    /**
+     * Used to keep track of all the commands available
```

Comment:
Very very formatting minor issue, but I noticed that the new JavaDocs do not end with a full stop. If i'm not wrong, all other JavaDocs sentences end with a full stop.